cognee/tasks/cleanup.py

```
@@ -0,0 +1,49 @@
+from datetime import datetime, timedelta
+from typing import Optional
+import uuid
+
+async def cleanup_unused_data(
+    days_threshold: int = 30,
+    dry_run: bool = True,
+    user_id: Optional[uuid.UUID] = None
```

Ruff (0.12.2)

Unused function argument: `user_id`

(ARG001)